src/pages/plot_features/plot_map.js

```diff
@@ -1,9 +1,10 @@
 import React, { forwardRef, useCallback, useEffect, useImperativeHandle, useRef, useState } from 'react';
```

Comment:
![Screenshot 2024-11-05 at 10 34 17 AM](https://github.com/user-attachments/assets/bf3849c6-c37a-4471-bd5b-6c20aa39e00a)
The increased space between zoom buttons and rotate buttons looks a little weird.